src/schema_formats/XML/GMP.xml.in

```diff
@@ -27174,6 +27174,16 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
     </description>
     <version>20.04</version>
   </change>
+  <change>
```

Comment:
Not sure why you're adding this.  We don't normally mention extensions, only changes and removals.

Reply:
Well, it moves the location of the solution/solution_type, which is a change.
But in fact the current situation is not documented here, so it looks like an addition.

I addressed the issue with another PR already.

Reply:
From a protocol perspective the PR doesn't really change anything, because the protocol does not specify the contents of NVT/TAGS.

If we're going to keep the CHANGE it should say that the solution method moved from the tags to a new element.  This way it's clearer that it's somehow not just an extension.

Also was solution type previously in the tags?  I don't see a CHANGE for that, but it should be consistent.

Reply:
It appears simpler to remove the entry, so I did.
The tags remains untouched yet, see general description about this PR.